AUTHORS

```diff
@@ -651,6 +651,7 @@ Contributors:
  zhaolong
  张昆
  练亮斌
+ powersagitar
```

Comment:
Don't add yourself to the authors list. It’s generated automatically and any manual changes will get lost.